corehq/apps/app_manager/templates/app_manager/v1/managed_app.html

```diff
@@ -110,12 +110,22 @@
                     <li class="edit-module-li {% ifequal module.id selected_module.id %}{% if not form %}
                         active{% endif %}{% endifequal %}"
                         data-index="{{ module.id }}">
-                        <a href="{% url "view_module" domain app.id module.id %}">
+                        <a href="{% url "view_module" domain app.id module.id %}"
+                            >
                             <i class="drag_handle"></i>
-                            {% if module.module_type == 'advanced' %}<i class="fa fa-flask appmanager-icon-type"></i>
-                            {% elif module.module_type == 'report' %}<i class="fa fa-bar-chart appmanager-icon-type"></i>
-                            {% elif module.module_type == 'shadow' %}<i class="fa fa-folder-open-o appmanager-icon-type"></i>
-                            {% else %}<i class="fa fa-folder-open"></i>{% endif %}
+                            <i
+                            {% if module.module_type == 'advanced' %}class="fa fa-flask appmanager-icon-type"
+                            {% elif module.module_type == 'report' %}class="fa fa-bar-chart appmanager-icon-type"
+                            {% elif module.module_type == 'shadow' %}class="fa fa-folder-open-o appmanager-icon-type"
+                            {% else %}class="fa fa-folder-open"{% endif %}
+
+                            {% if module.case_type and request|toggle_enabled:"SUPPORT" %}
```

Comment:
last time i tried this with form icons it was decided to go under support toggle. happy to take out though, though should ping @dimagi/ux and @dimagi/product